test-result-summary-client/src/utils/Utils.js

```diff
@@ -44,4 +44,4 @@ const getSHA = (type, javaVersion) => {
         }
     }
     return null;
-}
```

Comment:
There is no code change in this file. Please revert the format change

Reply:
I have not explicitly changed the format